### PR TITLE
Apply breakdown sum validation to QBS

### PR DESCRIPTION
**Pull request by dwyeradam**
### What is the context of this PR?
As a respondent
I want my answers validated before I submit
So that ONS does not contact me after submission

### How to review 
Enter a value for total number of employees.
The sum of the breakdown answers must then total less than or equal to that value. Otherwise a correct error message should be displayed.

### CODE REVIEW

**Comment by dwyeradam** [comment]
@shahi645 is there a reason the label isn't piped into the error message anymore? I have used a bespoke message to the same effect.

**Review by jonnyshaw89 — APPROVED** [review]

**Comment by shahi645** [comment]
@dwyeradam We went through the content me Charlotte and Will, we agreed to take it out

File: data/en/2_0001.json [hunk 1]
```diff
@@ -154,7 +154,28 @@
...
+                            ],
+                            "conditions": [
+                                "less than",
+                                "equals"
+                            ]
+                        },
+                        "validation": {
+                            "messages": {
+                                "TOTAL_SUM_NOT_LESS_THAN_OR_EQUALS": "Enter answers that add up to or are less than the Total number of employees, %(total)s"
+                            }
```
Comment (shahi645) [inline data/en/2_0001.json]: I think we just want to use the standard error messages @charlotte-hirst can you confirm?

File: data/en/2_0001.json [hunk 2]
```diff
@@ -154,7 +154,28 @@
...
+                            "answer_id": "number-of-employees-total",
+                            "answers_to_calculate": [
+                                "number-of-employees-male-more-30-hours",
+                                "number-of-employees-male-less-30-hours",
+                                "number-of-employees-female-more-30-hours",
+                                "number-of-employees-female-less-30-hours"
+                            ],
+                            "conditions": [
+                                "less than",
+                                "equals"
```
Comment (shahi645) [inline data/en/2_0001.json:171]: Also are we going with less than or equals?

File: data/en/2_0001.json (hunk 1 above)
Comment (charlotte-hirst) [reply data/en/2_0001.json]: yes standard message without piping of answer label please

File: data/en/2_0001.json (hunk 2 above)
Comment (charlotte-hirst) [reply data/en/2_0001.json:171]: its less than or equal for employment

File: data/en/2_0001.json (hunk 1 above)
Comment (jonnyshaw89) [reply data/en/2_0001.json]: @shahi645 @charlotte-hirst we need to make sure that offline conversations are updated in the specs

**Comment by dwyeradam** [comment]
@shahi645 Bespoke error message removed so ready for review please.

**Review by shahi645 — APPROVED** [review]
LGTM

**Pull request merged**